client/layout/sidebar/menu.jsx

```diff
@@ -2,9 +2,10 @@
  * External dependencies
  */
 import React from 'react';
+import classNames from 'classnames';
 
-const SidebarMenu = ( { children } ) => (
-	<li className="sidebar__menu">{ children }</li>
+const SidebarMenu = ( { children, className } ) => (
```

Comment:
It's the parent element inside `ExpandableSidebarMenu`, which passes in classes based on the toggle state:

```		const classes = classNames(
			this.props.className,
			{
				'is-toggle-open': !! this.props.expanded,
				'is-togglable': true
			}
		);```